docs/VehicleInfo/SubscribeVehicleData/index.md

```diff
@@ -101,6 +103,17 @@ SubscribeVehicleData
 ![SubscribeVehicleData](./assets/SubscribeVehicleData.jpg)
 |||
 
+|||
+Resumption of multiple applications with common subscriptions
+![SubscribeVehicleData](./assets/MultipleAppCommonSubscriptions.png)
+|||
+
+|||
+Error handling for subscriptions during parallel resumption of two applications
```

Comment:
I know the diagram is from the proposal, however it does not match the implementation.

In the implementation, SDL send subscribe data 1 and 2, and freeze the request of subscribe data 2,3,4. only when SDL get the response from HMI regarding data 1 and 2, SDL send one more frozen request.

Reply:
@yang1070  this diagram will be a bit different for different types of subscriptions. 
Freezing request to HMI logic is related only to vehicle data subscriptions, because one  subscribe vehicle data request contains many vehicle data items to subscribe, and each vd item should be processed separately, so to avoid dramatically complicated logic with simultaneous error handling of multiple Subscribe VD requests, "freezing" logic was implemented. 
The "freeze logic" is part of internal SDL implementation and will be described in https://github.com/smartdevicelink/sdl_core_guides/ in separate sequence diagramm

But all other resumption data (Add commands, submenus,etc ...) is actually resumed in parallel in the way this diagram shown.  

To avoid confusion we will replace "subscriptions" in this diagram to "resumption data".

Reply:
since this is SubscribeVehicleData asset, I believe replacing  "subscriptions" with "resumption data" wouldn't illustrate  "freezing" logic.
I suggest this change https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/224/commits/aa59dee82cee76cea6705c55e708b23883483f13